nf_core/pipeline-template/{{cookiecutter.name_noslash}}/bin/check_samplesheet.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 # TODO nf-core: Update the script to check the samplesheet
+# This script is based on the example at: https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_test_sispa.csv
```

Comment:
I've added the link here to the example. Just thinking about it, adding the example in assets might be confusing and might make people think that the sample sheet for the test data should be placed there, maybe.

Reply:
Thanks!